src/game.cpp

```diff
@@ -2228,7 +2264,13 @@ input_context game::get_player_input(std::string &action)
         }
         inp_mngr.set_timeout(-1);
     } else {
-        while (handle_mouseview(ctxt, action)) {};
+        inp_mngr.set_timeout(125);
+        while (handle_mouseview(ctxt, action)) {
+            if (action == "TIMEOUT" && has_user_turn_timeout_elapsed()) {
```

Comment:
handle_mouseview() returns true => action == "TIMEOUT" || ctxt.get_raw_input().get_first_input() == ERR. Simply going from that post-condition, it's possible that action != "TIMEOUT". It probably wouldn't make an observable difference either way.

I vaguely recall error events when mousing or clicking along the edge of the screen. I'd have to double check that though. That might have been for something unrelated to this code.